### ISSUE TITLE
Weird typedoc error breaks my serve - anyone?

### ISSUE THREAD

**Issue by spock123** [AngularClass/angular2-webpack-starter#799]
Hey guys,
- I have a project on which I've used this excellent starter as a base. Each time there are updates, I've updated my application - everything's been smooth.

But after updating to Typescript2.0 and @typings  I get some weird errors when I try to serve my app in development mode. During Webpack build, I get the following errors, after which my build halts completely without further messages.

```
`34% 4/10 build modules{ [Error: ENOENT: no such file or directory, open '/Users/spock/frontend/node_modules/typedoc/src/lib/tsd.d.ts']
  errno: -2,
  code: 'ENOENT',
  syscall: 'open',
  path: '/Users/spock/frontend/node_modules/typedoc/src/lib/tsd.d.ts' } 'Error: ENOENT: no such file or directory, open \'/Users/spock/frontend/node_modules/typedoc/src/lib/tsd.d.ts\'\n    at Error (native)'

```
When I look in the node_modules/typedoc folder, there isn't even a src folder!
Anyone have an idea of what the **** could be referencing typedoc source?

I've tried uninstall typedoc to no avail

Cheers

**Comment by abdulhaq-e** [comment]
I believe `typedoc` doesn't support TypeScript 2 yet.

**Comment by spock123** [comment]
Hi @abdulhaq-e  - that's true - but I don't even run typedoc.. and I'm getting same errors even if I uninstall typedoc..  this is so weird

**Comment by spock123** [comment]
Ok I found the error - and maybe it can be of interest to people upgrading this project by hand.

I turns out - that yes, typedoc does not support typescript2.0 yet.
So far so good - I removed typedoc, BUT  node_modules/.bin/typedoc was not removed in that process.

So: if you upgrade to TS2.0, make sure you remove tsdoc completely - ALSO in the node_modules/.bin folder.

Thanks guys, closing the case.
